crates/ef-testing/src/models/result.rs

```diff
@@ -11,19 +11,19 @@ use tracing::{error, info, warn};
 
 #[allow(clippy::cognitive_complexity)]
 pub(crate) fn log_execution_result(
-    result: TransactionExecutionResult<TransactionExecutionInfo>,
+    result: &TransactionExecutionResult<TransactionExecutionInfo>,
     case_name: &str,
     case_category: &str,
 ) {
     let case = format!("{}::{}", case_category, case_name);
     match result {
         TransactionExecutionResult::Ok(info) => {
-            if let Some(err) = info.revert_error {
+            if let Some(err) = info.revert_error.as_ref() {
                 warn!("{} reverted:\n{}", case, err.replace("\\n", "\n"));
             } else {
                 info!("{} passed: {:?}", case, info.actual_resources);
                 #[cfg(feature = "v0")]
-                if let Some(call) = info.execute_call_info {
+                if let Some(call) = info.execute_call_info.as_ref() {
```

Comment:
do we need this?

Reply:
yeah otherwise it won't compile 🤔 "moved" problems, I think otherwise it takes ownership

Reply:
cannot move out of `info.revert_error` as enum variant `Some` which is behind a shared reference